source/modes/GameEditorModeBase.cpp

```diff
@@ -132,6 +133,12 @@ GameEditorModeBase::~GameEditorModeBase()
     // Delete the potential pending event messages
     for (EventMessage* message : mEventMessages)
         delete message;
+
+    // Clear up any events and chat messages.
+    mRootWindow->getChild("GameChatWindow/GameChatText")->setText("");
```

Comment:
When there is conflict in git I choose the piece which has the code in it, that's my secret to be honest , and it seems that it does not do it's job very well ./